react/ProductSummary.js

```diff
@@ -46,14 +47,14 @@ class ProductSummary extends Component {
       badgeText,
       buyButtonText,
       hideBuyButton,
-      showButtonOnHover,
     } = this.props
 
     const product = this.props.product || createProduct()
+    const showButtonOnHover = this.props.showButtonOnHover && !isMobile
```

Comment:
I don't think it is necessary to add a dependency for this. You can try to use the media query classes, for example: `this.props.showButtonOnHover ? 'db' : 'db dn-ns'`

Reply:
The thing is that tablets screen sizes are not that small. That's why I wanted to do a mobile detection. Should I ignore the tablets fact?

Reply:
I still think this is an overkill but have no better solution, so leave it the way it is.